src/ecommerce.js

```diff
@@ -539,7 +540,7 @@ export default function Ecommerce(mpInstance) {
 
             baseEvent.CurrencyCode = mpInstance._Store.currencyCode;
             baseEvent.ShoppingCart = [];
-            baseEvent.CustomFlags = customFlags;
+            baseEvent.CustomFlags = extend(baseEvent.CustomFlags, customFlags);
```

Comment:
Can we use spread or the existing merge objects util? I'd like to get rid of the extend helper.